lib/pages/search/view/search_page.dart

```diff
@@ -0,0 +1,299 @@
+            Container(
+                child: Column(
+              children: [
+                const Image(
+                  image:
+                      AssetImage('assets/illustrations/undraw_chat_image.png'),
+                ),
+                Padding(
+                  padding: const EdgeInsets.fromLTRB(10, 10, 10, 10),
+                  child: Row(
+                    mainAxisAlignment: MainAxisAlignment.center,
+                    children: [
+                      Padding(
+                          padding: const EdgeInsets.only(right: 10),
+                          child: Text(S.current.messageAnotherQuestion)),
+                      Text(
+                        S.current.messageTalkToChatbot,
+                        style: TextStyle(
+                            color: Theme.of(context).accentColor,
+                            fontWeight: FontWeight.bold,
+                            fontSize: 15),
+                      )
+                    ],
+                  ),
+                )
+              ],
+            ))
+        ],
+      ),
+    );
+  }
+}
+
+class PeopleCircleList extends StatelessWidget {
+  const PeopleCircleList({this.people, this.query});
+
+  final List<Person> people;
+  final String query;
+
+  @override
+  Widget build(BuildContext context) {
+    return Container(
+      constraints: BoxConstraints.expand(
+        height: Theme.of(context).textTheme.headline4.fontSize * 1.1 + 80.0,
+      ),
+      padding: const EdgeInsets.all(10),
+      child: ListView.builder(
+        shrinkWrap: true,
+        itemCount: people.length,
+        scrollDirection: Axis.horizontal,
+        itemBuilder: (BuildContext context, int index) {
+          final List<String> name = people[index].name.split(' ');
+          return Container(
+              padding: const EdgeInsets.fromLTRB(0, 0, 30, 0),
+              child: GestureDetector(
+                onTap: () {
+                  showModalBottomSheet<dynamic>(
+                      isScrollControlled: true,
+                      context: context,
+                      builder: (BuildContext buildContext) => PersonView(
+                            person: people[index],
+                          ));
+                },
+                child: Column(children: <Widget>[
+                  const CircleAvatar(
+                    backgroundImage: NetworkImage(
+                        'https://identix.state.gov/qotw/images/no-photo.gif'),
+                  ),
+                  for (String n in name) Text(n),
+                ]),
+              ));
+        },
+      ),
+    );
+  }
+}
+
+class ClassesCircleList extends StatelessWidget {
+  const ClassesCircleList({this.classesHeader, this.query});
+
+  final List<ClassHeader> classesHeader;
+  final String query;
+
+  @override
+  Widget build(BuildContext context) {
+    if (classesHeader.isNotEmpty) {
+      final int nr = classesHeader.length > 3 ? 3 : classesHeader.length;
```

Comment:
This could just be nr = min(classHeaders.length, 3), this is not necessary at all, check my comment below.